zmq/asyncio.py

```diff
@@ -49,23 +49,35 @@ class _AsyncIO(object):
     _Future = Future
     _WRITE = selectors.EVENT_WRITE
     _READ = selectors.EVENT_READ
-    
+
     def _default_loop(self):
         return asyncio.get_event_loop()
 
 
-class ZMQSelector(selectors.BaseSelector):
+class ZMQSelector(selectors._BaseSelectorImpl):
```

Comment:
Agree. I went for the private class because it does all that is needed so was the simplest/fastest way to get a first version working.
No public APIs, would have to copy/reimplement code from selectors.py. No big deal, the whole implementation of _BaseSelectorImpl is ~ 90 lines of code including comments.